config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require "sidekiq/web"
+
+if ENV.key?("VCAP_SERVICES")
```

Comment:
How is this configured locally?

Reply:
If nothing is configured it defaults to http://127.0.0.1:6379/, which is the port redis defaults to when you run `redis-server` locally as well, might stick that in a comment or the readme.